xbmc/Util.h

```diff
@@ -70,6 +69,8 @@ class CUtil
   static int GetDVDIfoTitle(const std::string& strPathFile);
 
   static bool IsPicture(const std::string& strFile);
+  /// Get resolved filesystem location of splash image
+  static std::string GetSplashPath();
```

Comment:
could you please add a valid doxy comment

Reply:
What are you missing here?

Reply:
```c++
/*!
 \brief Return the resolved filesystem ...
 \return the path ....
 */
```

Reply:
Isn't that the same but longer? `///` is picked up by doxygen and used for the `brief`.

Reply:
sure, but be consistent would be appreciated and you could differ the brief from the return description.

Reply:
`///` is already used all over the place. And honestly I don't know what more I could say in `\return`. It would be just the same information as in the brief but worded differently, which is not helpful.

Reply:
ok leave it if that is such a big thing.